sdk/src/main/java/io/horizen/account/state/ExecutionRevertedException.java

```diff
@@ -1,23 +1,25 @@
 package io.horizen.account.state;
 
+import org.web3j.utils.Numeric;
+
 /**
  * Message processing failed, also revert-and-keep-gas-left.
  */
 public class ExecutionRevertedException extends ExecutionFailedException {
-    public final byte[] revertReason;
+    public final byte[] returnData;
```

Comment:
Considering how it is used, maybe it is better to save directly the hex string instead of byte[]

Reply:
For me the current usage is just a particular case to "print" the field, but in general it is expected to be in a raw format and can be used by native/evm smart contract processors.
For example, later we will introduce the interoperability between the contracts, so the caller-contract may require to process the `returnData` field from the delegated-contract exception.

Reply:
OK, that makes sense.